monitors/SyntheticPegMonitor.js

```diff
@@ -0,0 +1,101 @@
+const { createFormatFunction, createEtherscanLinkMarkdown } = require("../common/FormattingUtils");
+const { createObjectFromDefaultProps } = require("../common/ObjectUtils");
+
+class SyntheticPegMonitor {
+  /**
+   * @notice Constructs new Liquidator bot.
+   * @param {Object} logger Module used to send logs.
+   * @param {Object} web3 Instance of a web3 client provided by the class that initiates the monitor module.
+   * @param {Object} uniswapPriceFeed Module used to query the current uniswap token price.
+   * @param {Object} medianizerPriceFeed Module used to query the current crypto watch token price.
+   * @param {Object} [config] Contains fields with which constructor will attempt to override defaults.
+   */
+  constructor(logger, web3, uniswapPriceFeed, medianizerPriceFeed, config) {
+    this.logger = logger;
+
+    // Instance of price feeds used to check for deviation of synthetic token price.
+    this.uniswapPriceFeed = uniswapPriceFeed;
+    this.medianizerPriceFeed = medianizerPriceFeed;
+
+    this.web3 = web3;
+
+    // Contract constants
+    // TODO: replace this with an actual query to the collateral currency symbol
+    this.collateralCurrencySymbol = "DAI";
+    this.syntheticCurrencySymbol = "UMATEST";
```

Comment:
Config?

Reply:
We _could_ do this but right now all the bots have this same hardcoding. I'll leave this for a separate PR. It is already documented in [this](https://github.com/UMAprotocol/protocol/issues/1417) issue.

Reply:
SGTM